src/backend/src/utils/slack.utils.ts

```diff
@@ -37,3 +37,22 @@ export const sendSlackUpcomingDeadlineNotification = async (workPackage: WorkPac
 
   await sendMessage(LEAD_CHANNEL_SLACK_ID, fullMsg);
 };
+
+export const sendSlackRequestedReviewNotification = async (slackId: string, changeRequest: ChangeRequest): Promise<void> => {
+  if (process.env.NODE_ENV !== 'production') return; // don't send msgs unless in prod
+
+  const { requestedReviewers } = changeRequest;
+  const slackIds = requestedReviewers.map(async (reviewer) => await getUserSlackId(reviewer.userId));
+
+  Promise.all(slackIds).then(async (slackIds) => {
```

Comment:
woah whats going on here, i think you got a little confused with whats happening, i think all we want to do is just send the given user with the slack id a message that theyve been assigned to review this CR. so really all you need is just the the crString, because this should be sending it to people individually. so instead of all these loops, just take the crString to sendmessage code and delete everything else. Id also rephrase the message to be, You have been assigned to review CR #${changeRequest.crId}: ${crString}. 
id also rename crString to be changeRequestLink